### ISSUE TITLE
Import SMDs?

### ISSUE THREAD

**Issue by vaqxai** [Ray1235/SourceMayaTools#5]
Is there a sister-project of this to allow for importing of SMDs (To modify existing models?)

**Comment by TheEnbyWitch** [comment]
Not really, the only Maya plugin that I know of that supports SMD import is MESA, which has broken scale and weights in imports, so I'd recommend using the blender plugin (edited to import at a 2.54 scale) and export an FBX from that.

Importing SMDs is still on my wishlist though, so when I get the the time and motivation to do it, I'll do it.

**Comment by vaqxai** [comment]
Thanks, but i stumbled on another problem while trying to export animated
smds, the scale is way off. What did I do wrong?

**Comment by TheEnbyWitch** [comment]
Oh yeah, forgot to mention that MESA also imports stuff at the wrong scale.

What's happening exactly?
